scripts/lint.sh

```diff
@@ -15,6 +15,14 @@ FAILED=0
 IGNORED_FILES="uv\.lock|docs\/CARS.md"
 IGNORED_DIRS="^third_party.*|^msgq.*|^msgq_repo.*|^opendbc.*|^opendbc_repo.*|^cereal.*|^panda.*|^rednose.*|^rednose_repo.*|^tinygrad.*|^tinygrad_repo.*|^teleoprtc.*|^teleoprtc_repo.*"
 
+function check_banned_patterns() {
+  BANNED_PATTERN='\.vEgo(?!Stopping|Starting|Raw|Cluster)|\.aEgo'
+  if grep -r -P "$BANNED_PATTERN" --include='*.py' selfdrive/car; then
+    echo 'Found banned pattern!'
+    exit 1
+  fi
+}
+
```

Comment:
@maxime-desroches is this something we want in the lint tool?

Reply:
If you think it might be useful while you switch everything over to opendbc, then we can add it. We might want to put these custom lint rules in separate files since we will also need a rule to enforce `python3` instead of `python` in shebangs and I would prefer to keep `lint.sh` as simple as possible. 

Maybe we can have a `lint/` subdirectory will all the custom rules we need?

Reply:
That's fine, this isn't needed once I merge dataclasses, as linting will catch these errors. `lint/` directories sound good.